Source/WebCore/css/CSSValue.h

```diff
@@ -71,81 +71,81 @@ class CSSValue {
-    bool isImageSetOptionValue() const { return m_classType == ImageSetOptionClass; }
-    bool isImageSetValue() const { return m_classType == ImageSetClass; }
-    bool isImageValue() const { return m_classType == ImageClass; }
-    bool isInsetShape() const { return m_classType == InsetShapeClass; }
-    bool isLineBoxContainValue() const { return m_classType == LineBoxContainClass; }
-    bool isLinearGradientValue() const { return m_classType == LinearGradientClass; }
-    bool isLinearTimingFunctionValue() const { return m_classType == LinearTimingFunctionClass; }
-    bool isNamedImageValue() const { return m_classType == NamedImageClass; }
-    bool isOffsetRotateValue() const { return m_classType == OffsetRotateClass; }
-    bool isPair() const { return m_classType == ValuePairClass; }
-    bool isPath() const { return m_classType == PathClass; }
-    bool isPendingSubstitutionValue() const { return m_classType == PendingSubstitutionValueClass; }
-    bool isPolygon() const { return m_classType == PolygonClass; }
-    bool isPrefixedLinearGradientValue() const { return m_classType == PrefixedLinearGradientClass; }
-    bool isPrefixedRadialGradientValue() const { return m_classType == PrefixedRadialGradientClass; }
-    bool isPrimitiveValue() const { return m_classType == PrimitiveClass; }
-    bool isQuad() const { return m_classType == QuadClass; }
-    bool isRadialGradientValue() const { return m_classType == RadialGradientClass; }
-    bool isRayValue() const { return m_classType == RayClass; }
-    bool isRect() const { return m_classType == RectClass; }
-    bool isRectShape() const { return m_classType == RectShapeClass; }
-    bool isReflectValue() const { return m_classType == ReflectClass; }
-    bool isScrollValue() const { return m_classType == ScrollClass; }
-    bool isShadowValue() const { return m_classType == ShadowClass; }
-    bool isShape() const { return m_classType == ShapeClass; }
-    bool isShapeSegment() const { return m_classType == ShapeSegmentClass; }
-    bool isSpringTimingFunctionValue() const { return m_classType == SpringTimingFunctionClass; }
-    bool isStepsTimingFunctionValue() const { return m_classType == StepsTimingFunctionClass; }
-    bool isSubgridValue() const { return m_classType == SubgridClass; }
-    bool isTransformListValue() const { return m_classType == TransformListClass; }
-    bool isUnicodeRangeValue() const { return m_classType == UnicodeRangeClass; }
-    bool isValueList() const { return m_classType == ValueListClass; }
-    bool isVariableReferenceValue() const { return m_classType == VariableReferenceClass; }
-    bool isViewValue() const { return m_classType == ViewClass; }
-    bool isXywhShape() const { return m_classType == XywhShapeClass; }
-    bool isPaintImageValue() const { return m_classType == PaintImageClass; }
+    bool isAnchorValue() const { return classType() == ClassType::Anchor; }
```

Comment:
Calling `classType()` for these in unnecessary, we can still just compare using `m_classType` if you change its type from an `unsigned` to `ClassType`